packages/now-cli/src/util/output/logo.ts

```diff
@@ -1 +1 @@
-export default (process.platform === 'win32' ? 'Δ' : '𝚫');
+export default process.platform === 'win32' ? '▲' : '▲';
```

Comment:
```suggestion
export default '▲';
```
Are those two triangles the same?

Also, yes this character exists in Linux:
![Screenshot from 2020-01-13 13-05-41](https://user-images.githubusercontent.com/11590024/72280204-cc630800-3605-11ea-8103-c3eae3b14724.png)